pkg/controller/petset/pet.go

```diff
@@ -216,6 +220,7 @@ func (p *apiServerPetClient) Update(pet *pcb, expectedPet *pcb) (updateErr error
 		if p, getErr = pc.Get(pod.Name); getErr != nil {
 			return getErr
 		}
+		time.Sleep(retrySleepTime)
```

Comment:
I intended to sleep between each update retry. But I think you are right. Thanks!